### ISSUE TITLE
nuget

### ISSUE THREAD

**Issue by jeremysimmons** [bobvanderlinden/sharpfilesystem#1]
I'm interested in using this project, but would like a nuget package.
would you accept a pull request for a nuget package?

**Comment by FrozenCow** [comment]
Yes! A PR is very much appreciated.

**Comment by ronnieoverby** [comment]
Ditto

**Comment by jeremysimmons** [comment]
epic fail. I can't believe I never took you up on the offer for this @FrozenCow

**Comment by FrozenCow** [comment]
@jeremysimmons No problem, PRs are appreviated any and all times! ;)

I've just updated the project a bit to work correctly with VS2015. I've also removed the original References directory and used nuget to retrieve external dependencies (zip/7zip dependencies). That should make things a bit nicer.

I have little experience (none to be exact) with publishing Nuget packages to nuget.org, but I could give it a go. Let me know if you want to package it or if you have any suggestions for packaging?
